docs/migration.md

```diff
@@ -251,6 +226,97 @@ consider additional positioning prop support on a case-by-case basis.
 - Migrate to `@zendeskgarden/container-utilities` and `@zendeskgarden/react-theming` to continue
   receiving updates.
 
+### Deprecated components
+
+The following subcomponents have been renamed to streamline imports and improve
+affinity with their parent components.
+
+<!-- markdownlint-disable MD024 -->
+
+#### @zendeskgarden/react-chrome
+
+- Subcomponent exports have been deprecated and will be removed in a future major version. Update
+  to subcomponent properties:
```

Comment:
[nit] Should these sentences move to the top intro to avoid repetitions.
```suggestion

```

Reply:
My imperession is that the top paragraph covers this statement.